spec/controllers/graphql/mutations/freak_create_spec.rb

```diff
@@ -12,6 +12,12 @@ module Graphql
       }
     end
 
-    it { is_expected.to match_response_for(mutation: :freak_create, sample: :freakCreate) }
+    context 'when role exists' do
```

Comment:
This context would make one think about `when the roles does not exist` as well. Is that a valid case?

If not, I doubt this context makes sense to exist.